dcmorales/digital-keyboard-tunes
# Feature: highlight notes based on selected key and octave

## Description
Added highlighting to the selected scale. It is currently red, to be replaced later. The notes that are highlighted are based on the selections made. Chromatic will include all notes in that octave and highlight the keys both in the keyboard-selected and in the keyboard-full. The major option has also been added to the scales dropdown.  When the scales are played, the notes appear active. The context has been updated to provide the notes in the selected octave as well as the notes in the selected scale.

## Steps to test / reproduce
On the home page, notice all keys in the keyboard-selected are highlighted (due to chromatic being the default setting). Update the scale to major and the highlighted keys will reflect the update.

## Types of changes



- [ ] Task (non-breaking change that is not a bug or feature)

## Reminders



- [x] I have tested that my code builds successfully
- [x] I have pulled the latest version of this branch's base branch (i.e. `main`)
- [x] I have updated the README or inline docs if needed
- [x] I have added a short, descriptive title to this PR

## Other comments
